public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_monitor/url_builder.ts

```diff
@@ -57,7 +57,7 @@ export default class UrlBuilder {
       );
     }
 
-    return `${baseUrl}${resourceUri}/providers/microsoft.insights/metricNamespaces?api-version=${apiVersion}`;
+    return `${baseUrl}${resourceUri}/providers/microsoft.insights/metricNamespaces?region=global&api-version=${apiVersion}`;
```

Comment:
another small change that affects things outside the template variable. The `region` parameter is optional but it's necessary when the resourceGroup is not specified, this allows to list all metric namespaces in a subscription.